templates/cookie-notice.php

```diff
@@ -1,36 +1,23 @@
 <?php
 
 namespace Netzstrategen\CoreStandards;
-
 ?>
 
-<style>
-  #cookie-notice {
-    position: fixed;
-    bottom: 0;
-    left: 0;
-    right: 0;
-    z-index: 100000;
-    padding: 0.5em;
-    text-align: center;
-    background-color: #000;
-    color: #fff;
-  }
-
-  <?= do_action('core_standards/cookie_notice/css_extend') ?>
-</style>
-
-<div id="cookie-notice" role="banner">
-  <span><?= __('This website uses cookies to give you the best possible service. By continuing to use the website, you agree to the use of cookies.', Plugin::L10N) ?></span>
-  <a href="#" id="cookies-accept" class="btn"><?= __('OK') ?></a>
-  <a href="<?= apply_filters('core_standards/cookie_notice/policy_link', '/privacy') ?>" target="_self" class="btn"><?= __('Learn more', Plugin::L10N) ?></a>
+<div class="cookie-notice" id="cookie-notice" role="banner">
+  <div class="cookie-notice__body">
+    <?= __('This website uses cookies to give you the best possible service. By continuing to use the website, you agree to the use of cookies.', Plugin::L10N) ?>
+    <a class="cookie-notice__link" href="<?= apply_filters('core_standards/cookie_notice/policy_link', '/privacy') ?>" target="_self"><?= __('Learn more', Plugin::L10N) ?></a>
```

Comment:
As just discussed in person, the privacy policy link should be incorporated into the text.

Let's make sure to create a task for this and resolve it before this change gets deployed to further sites, as this would cause lots of unnecessary update work otherwise.

Reply:
Initial text suggestion after doing a quick review of prominent German websites:

```diff
     <?= __('This website uses cookies to give you the best possible service.
-            By continuing to use the website, you agree to the use of cookies.', Plugin::L10N) ?>
+            By continuing to use it, you agree to our <a href="%s">privacy policy</a>.', Plugin::L10N) ?>
```

_"Diese Website verwendet Cookies, um den bestmöglichen Service zu bieten. Mit der weiteren Nutzung akzeptieren Sie unsere Datenschutzrichtlinie."_

The most problematic aspect is the German translation, so the follow-up task should focus on that.